packages/api/pages/api/languages/[code]/members/index.ts

```diff
@@ -102,13 +107,11 @@ export default createRoute<{ code: string }>()
             providerUserId: email,
             password: null,
           },
-          attributes: {
-            email,
-          },
+          attributes: {},
         });
 
         const token = randomBytes(12).toString('hex');
-        await auth.createKey(user.userId, {
+        await auth.createKey(user.id, {
           type: 'single_use',
           providerId: 'email-verification',
```

Comment:
If you invite a user from a language, they don't get prompted to enter their name. Maybe changing this to `providerId: 'invite-verification'` to  match `api/users/index.ts` would work?

It looks like there is a good amount of duplicate code between here and there. We could avoid some future inconsistency problems if we moved the duplicate code into a helper function.

Reply:
Ok, `providerId: 'invite-verification'` isn't the right fix. I tried it.